source/balances/contracts/BalanceChecker.sol

```diff
@@ -146,8 +146,8 @@ contract BalanceChecker is Ownable {
    * @notice Allow owner to withdraw ether from contract
    */
   function withdraw() public onlyOwner {
-    (bool success, ) = address(owner()).call.value(address(this).balance)("");
-    require(success, "ETH_WITHDRAW_FAILED");
+    address payable payableOwner = address(uint160(owner()));
+    payableOwner.transfer(address(this).balance);
```

Comment:
Is this an update due to `0.5.16` changes?

Reply:
I actually did this while trying to figure out where the invalid opcode error was coming from (that I solved using `evmVersion: 'petersburg',`

I'll revert this to what it was

Reply:
Done